rotkehlchen/data_migrations/migrations/migration_11.py

```diff
@@ -42,10 +47,40 @@ def _get_optimism_transaction_fees(rotki: 'Rotkehlchen', progress_handler: 'Migr
         )
 
 
+def _detect_arbitrum_one_accounts(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:  # noqa: E501
+    """It duplicates accounts if active at arbitrum one only. The reason is to start the new
+    release with arbitrum one already detected"""
+    # Check updates for spam assets. This happens before accounts detection to avoid
+    # detecting accounts that only have spam assets. We also check for rpc nodes updates
+    # to avoid an edge case where a call to rpc nodes happens before the rpc node update task
+    # has finished.
+    progress_handler.new_step('Fetching new spam assets and rpc data info')
+    rotki.data_updater.check_for_updates(updates=[UpdateType.SPAM_ASSETS, UpdateType.RPC_NODES])
+
+    with rotki.data.db.conn.write_ctx() as write_cursor:
+        rotki.data.db.add_external_service_credentials(
+            write_cursor=write_cursor,  # add temporary etherscan arbitrum one key
+            credentials=[ExternalServiceApiCredentials(
+                service=ExternalService.ARBITRUM_ONE_ETHERSCAN,
+                api_key=ApiKey('VQUFYKKJR4RK8HFHYIJ9I93FIUVP44TN99'),  # same one in tests
+            )])
+
+    rotki.chains_aggregator.detect_evm_accounts(progress_handler)
+    # remove temporary etherscan arbitrum one key
+    rotki.data.db.delete_external_service_credentials([ExternalService.ARBITRUM_ONE_ETHERSCAN])
+
+
 def data_migration_11(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:
     """
     Introduced at v1.30.0
     """
     log.debug('Enter data_migration_11')
-    _get_optimism_transaction_fees(rotki, progress_handler)
+    with rotki.data.db.conn.read_ctx() as cursor:
+        cursor.execute('SELECT tx_hash, identifier FROM evm_transactions WHERE chain_id=10')
+        txhash_to_id = dict(cursor)
```

Comment:
can you add a comment mentioning that this is done here to count the total number of steps?